09-features.md

```diff
@@ -31,6 +31,16 @@ These flags may only be used in the `init` message:
 
 There are currently no `globalfeatures` flags.
```

Comment:
```suggestion
```

Reply:
Shall we just merge the two sections (`node_announcement` and `globalfeatures` flags?)

Reply:
I think it would make sense indeed, they describe the same thing.
Right now the spec mentions `globalfeatures` so I'd keep this section instead of the `node_announcement` one.
I'd let the feature bit unification PR deal with the re-work on that file to clarify local/global/node_announcement.